src/components/DataSubmissions/DataUpload.tsx

```diff
@@ -122,7 +119,7 @@ export const DataUpload: FC<Props> = ({ submission }: Props) => {
         Download Configuration File
       </StyledDownloadButton>
     );
-  }, [submission?.dataType, user?.role, collaborator]);
+  }, [submission, user, collaborator]);
```

Comment:
Since we're watching `submission`, do we still need to watch `collaborator` too?

```suggestion
  }, [submission, user]);
```

Reply:
Thanks for catching this, collaborator variable was only used here as well so I just went ahead and removed it entirely in [49843c4](https://github.com/CBIIT/crdc-datahub-ui/pull/573/commits/49843c4c5cf16dab4dd2009694927b49dc9aae16).